From f25c5fdb94bdf382c69e7891713f6de065836b0d Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 28 Nov 2019 18:04:24 +0800
Subject: [PATCH] ARM: rockchip_defconfig: add security configs

Sync with floral_defconfig
CONFIG_BUG_ON_DATA_CORRUPTION
CONFIG_SCHED_STACK_END_CHECK

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I97164085a28468667ae69b84632c3bd70c188614
---
 arch/arm/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 48be2a0ab568..0072825b66ba 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -753,5 +753,7 @@ CONFIG_SOFTLOCKUP_DETECTOR=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=10
 CONFIG_PANIC_TIMEOUT=5
 CONFIG_SCHEDSTATS=y
+CONFIG_SCHED_STACK_END_CHECK=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_ENABLE_DEFAULT_TRACERS=y
+CONFIG_BUG_ON_DATA_CORRUPTION=y
-- 
2.35.3

